tests/test_backend.py

```diff
@@ -674,6 +675,7 @@ def tearDown(self):
         django_redis.cache.DJANGO_REDIS_IGNORE_EXCEPTIONS = self._orig_setting
 
     def test_get(self):
+        assert self.cache._ignore_exceptions is True
```

Comment:
The test suite is using unittest style asserts, not the assert keyword. What do think about using `self.assertIs(self.cache._ignore_exceptions, True)` to remain consistent.

Does the test `DjangoOmitExceptionsTests` need to be updated too? It is also missing the `OPTIONS` key and doesn't have this assert.